integration_test/third_party_apps_data/agent/ops-agent/linux/supported_applications.txt

```diff
@@ -1,11 +1,9 @@
 activemq
 apache
 cassandra
-hadoop
```

Comment:
Should we add validation to enforce alphabetical order in this file, to prevent this from happening in the future?

Reply:
I don't see much need for alphabetical order enforcement. Duplicate entry enforcement on the other hand... it's a good idea, I'll consider it for a follow up PR.

Reply:
I split out the validation into https://github.com/GoogleCloudPlatform/ops-agent/pull/438 so that it can be reviewed separately.